tests/PHPStan/Analyser/data/array-functions.php

```diff
@@ -178,4 +178,16 @@
 $fooArray = ['foo'];
 $poppedFoo = array_pop($fooArray);
 
+/** @var int[] $integers */
+$sumInts = array_sum($integers);
```

Comment:
Please use the modern way of testing this, look at `NodeScopeResolver::testFileAsserts`.